apps/chess-training-app/src/components/training/ChessBoard.tsx

```
@@ -0,0 +1,261 @@
+            onClick={() => setPromotionMove(null)}
+            onContextMenu={(e) => {
+              e.preventDefault()
+              setPromotionMove(null)
+            }}
+            style={{
+              position: 'absolute',
+              inset: 0,
+              backgroundColor: 'rgba(0,0,0,0.3)',
+              zIndex: 1000,
+            }}
+          />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)